src/tools/chip-cert/CertUtils.cpp

```diff
@@ -1112,121 +1145,126 @@ CHIP_ERROR MakeCertTLV(CertType certType, const ToolChipDN * subjectDN, X509 * c
         writer.PutBytes(ContextTag(kTag_EllipticCurvePublicKey), subjectPubkey, chip::Crypto::CHIP_CRYPTO_PUBLIC_KEY_SIZE_BYTES));
 
     // extensions
-    ReturnErrorOnFailure(writer.StartContainer(ContextTag(kTag_Extensions), kTLVType_List, containerType2));
+    if (!useCompactIdentityFormat)
```

Comment:
This block of `if (!useCompactIdentityFormat)` pattern could have been used above as well where there are many new `if (!useCompactIdentityFormat && ` individual new compound conditions.

Reply:
Done